cmd/apiserver/apiserver.go

```diff
@@ -32,6 +32,7 @@ import (
 	// Please do not remove.
 	_ "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/install"
 
+	_ "github.com/kubernetes-incubator/service-catalog/cmd/apiserver/app"
```

Comment:
This is an anonymous import that exists solely to have another anonymous import call an `init()`.